import.py

```diff
@@ -0,0 +1,44 @@
+import os
+
+import csv
+from sqlalchemy import create_engine
+from sqlalchemy.orm import scoped_session, sessionmaker
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy import Column, Integer, String,DateTime,exists,Sequence
+from flask import Flask
+from flask_session import Session
+from flask_sqlalchemy import SQLAlchemy
+
+
+# Import table definitions.
+
+app1 = Flask(__name__)
+
+# Tell Flask what SQLAlchemy databas to use.
+app1.config["SQLALCHEMY_DATABASE_URI"] = os.getenv("DATABASE_URL")
```

Comment:
you are already having create.py, that needs to do all these. why are you again writing in some files.

Reply:
Hello Mam,
 They mentioned to write the table separately in assignment task on NGlms.
Hence I rewrote the code to create table.

Reply:
yes, get the instance of db from application.py and insert the books.

Reply:
hello mam, made the necessary changes. pls check. tq